Get SchwiftyTests/FormattingHelperTests.swift

```diff
@@ -0,0 +1,455 @@
+//
+//  FormattingHelperTests.swift
+//  GetSchwifty
+//
+//  Created by Lennart Wisbar on 17.05.17.
+//  Copyright © 2017 Benjamin Herzog. All rights reserved.
+//
+
+//  MARK: Test Naming Rule
+//  test_Action_Expectation(_Condition)
+
+import XCTest
+@testable import Get_Schwifty
+
+class FormattingHelperTests: XCTestCase {
+    
+    let textView = UITextView()
+    
+    override func setUp() {
+        super.setUp()
+    }
+    
+    override func tearDown() {
+        // Put teardown code here. This method is called after the invocation of each test method in the class.
+        super.tearDown()
+    }
```

Comment:
I see. I just left them there from the template, because I was unsure if they were needed.

Yes, initializing a new text view before each test seems safer.